bench/node_modules

```diff
@@ -0,0 +1 @@
+../node_modules/
```

Comment:
Not sure how useful will be this bench project might be in future, we can also remove it too.

This file is result of running `ln -s ../node_modules/ ./node_modules` in `bench` folder, so we don't mix bench scripts with root project.